### ISSUE TITLE
RCS1045: Debatable behavior

### ISSUE THREAD

**Issue by krisztiankocsis** [JosefPihrt/Roslynator#671]
**Product and Version Used**: Roslynator.Analyzers 2.3.0 (NuGet)

**Steps to Reproduce**:
```csharp
private static readonly PlatformID Platform = Environment.OSVersion.Platform;
```

**Actual Behavior**:
RCS1045: Reports this line if enabled.

**Expected Behavior**:
`const` and `static readonly` are "almost" the same, we can't use const in many cases. Consts are not reported, but static readonly members yes, I think they should not.

**Comment by JosefPihrt** [comment]
Fixed with ffb3b97c4e4e8ddb018be006ce240236c0fd82e1
